clusters/nerc-ocp-infra/vault/application.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: vault
+spec:
+  project: vault
```

Comment:
There is no `vault` project in argocd; this should be `default`.

Reply:
Updated to default